### PR TITLE
[EDX-498]: Code block is being weird when there is just one language

### PR DESCRIPTION
**Pull request by snikidev**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

A PR description indicating the purpose of the PR.

* [Jira ticket](https://ably.atlassian.net/browse/EDX-498).

## Review

Bug is visible on quick start guide page with python selected

`/docs/quick-start-guide?lang=python`

### CODE REVIEW

File: src/components/Menu/LanguageNavigation/LanguageNavigation.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const LanguageNavigation = ({ items, localChangeOnly, selectedLanguage, onSelect
   const shouldUseLocalChanges = localChangeOnly && !!onSelect;
   const onSelectChange = shouldUseLocalChanges ? onSelect : changePageOnSelect(pageLanguage);
 
-  return (
+  return items.length >= 2 ? (
```
Comment (MymmiJ) [inline src/components/Menu/LanguageNavigation/LanguageNavigation.tsx:54]: Can we also filter out empty items?

**Review by MymmiJ — CHANGES_REQUESTED** [review]
Can you get tests passing and filter items so that empty results aren't added to the Language Navigation?

**Review by aralovelace — COMMENTED** [review]
The test is also failing :(

**Review by aralovelace — APPROVED** [review]

File: src/components/Menu/LanguageNavigation/LanguageNavigation.tsx (hunk 1 above)
Comment (MymmiJ) [reply src/components/Menu/LanguageNavigation/LanguageNavigation.tsx:54]: Oh my bad! yes sorry

**Review by MymmiJ — APPROVED** [review]
Nice one, sorry about misleading feedback :+1:

**Pull request closed without merging**